src/DependencyInjection/Configuration.php

```diff
@@ -52,12 +52,15 @@ public function getConfigTreeBuilder(): TreeBuilder
                     ->useAttributeAsKey('name')
                     ->arrayPrototype()
                         ->children()
-                            ->enumNode('engine')->values(['chroma-db', 'azure-search'])->isRequired()->end()
+                            ->enumNode('engine')->values(['chroma-db', 'azure-search', 'mongodb'])->isRequired()->end()
                             ->scalarNode('collection_name')->end()
                             ->scalarNode('api_key')->end()
                             ->scalarNode('endpoint')->end()
                             ->scalarNode('index_name')->end()
                             ->scalarNode('api_version')->end()
+                            ->scalarNode('database_name')->end()
+                            ->scalarNode('ur')->end()
+                            ->scalarNode('vector_field_name')->end()
```

Comment:
In a follow up PR; we can add validate calls to ensure what is needed for which engine